src/expr.rs

```diff
@@ -1893,24 +1893,24 @@ fn rewrite_assignment(
     context: &RewriteContext<'_>,
     lhs: &ast::Expr,
     rhs: &ast::Expr,
-    op: Option<&ast::BinOp>,
+    op_span: Span,
     shape: Shape,
 ) -> Option<String> {
-    let operator_str = match op {
-        Some(op) => context.snippet(op.span),
-        None => "=",
-    };
-
     // 1 = space between lhs and operator.
+    let operator_str = context.snippet(op_span);
```

Comment:
might be better to swap line 1900 and 1899 so that the comment (`// 1 = space between lhs and operator.`) is closer to the line it's meant to comment

Reply:
Changed.